docs/pages/blog/turbo-1-1-0.mdx

```diff
@@ -0,0 +1,143 @@
+---
+title: Turborepo 1.1
+date: 2022/01/28
+description: Turborepo 1.1 introduces automatic migrations, turbo.json configuration, environment variable dependencies, partial Yarn v2 support, and more!
+tag: web development
+ogImage: /images/blog/turbo-1-1-0/twitter-card.png
+type: post
+authors: 
+  - jaredpalmer
+  - becca__z
+  - gaspargarcia_
+  - gsoltis
+---
+
+Since releasing Turborepo v1.0 in mid-December, we've seen incredible adoption:
+
+- 5.4k+ GitHub Stars
+- 60k+ weekly NPM downloads
+- 65+ OSS contributors
+- In production at [Vercel](https://github.com/vercel/next.js), [AWS](https://github.com/aws-amplify/amplify-ui), Paypal, [Twilio](https://github.com/twilio-labs/function-templates), [Contentful](https://github.com/contentful/forma-36), [Miro](https://github.com/miroapp/app-examples), [Framer](https://github.com/framer/motion), [Discord.js](https://github.com/discordjs/discord.js), [Rocket.chat](https://github.com/RocketChat/Rocket.Chat.Fuselage), [Astro.build](https://github.com/withastro/astro)
+- 575+ members of the [Turborepo Community Discord](https://turborepo.org/discord).
+
+![Weekly NPM downloads of `turbo`](/images/blog/turbo-1-1-0/turborepo-npm-downloads.png)
+
+We're further improving build performance and caching with Turborepo v1.1, featuring:
+
+- [**Automatic Migrations:**](#automatic-migrations) Official idempotent migration scripts to assist with upgrading.
+- [**`turbo.json` Support:**](#turbojson-support) Turborepo configuration now lives in its own file.
+- [**Faster Package Manager Detection:**](#faster-package-manager-detection) Turborepo now respects the `packageManager` key in the root `package.json`.
+- [**Environment Variable Dependencies:**](#environment-variable-dependencies) Define how environment variables impact global and task-specific caching.
+- [**Partial Support for Yarn v2+:**](#partial-yarn-v2v3-support) Support for yarn v2+ with [`nodeLinker: "node-modules"`](https://yarnpkg.com/configuration/yarnrc#nodeLinker).
```

Comment:
```suggestion
- [**Partial Support for Yarn v2+:**](#partial-yarn-v2v3-support) Support for yarn v2+ with `nodeLinker: "node-modules"`.
```

Making the only link here link to the section, rather than externally.